cwd/app.js

```diff
@@ -273,6 +273,19 @@
       characterText.textContent = view.message;
     }
 
+    // Testing functionality of length based font sized
+    let messageLength = characterText.innerText.trim().length;
+
+    if (messageLength < 70) {
+      characterText.style.fontSize = (.020 * document.getElementById('svg-wrap').width.baseVal.value) + 'px';
```

Comment:
maybe assign to a local variable first then set `characterText.style.fontSize` at end? may improve readability

Reply:
Also save `document.getElementById('svg-wrap').width.baseVal.value` into local `const` to not write so much repetitive code

Reply:
local variable could then append `px` only at the last save, saving code written